integration_tests/integration_tests.feature

```diff
@@ -91,38 +91,39 @@ Feature: Run integration tests
      When we run the d3a simulation on console with default_2a
      Then aggregated result files are exported
 
-  Scenario: Grid fees are calculated based on the original offer rate
```

Comment:
Why do these need to be disabled ? Can you explain how does this story affects the grid fees ?

Reply:
Sorry, you are right, re-adding it

Reply:
Please also delete the comment then. Thanks!